primitives/authority-discovery/src/lib.rs

```diff
@@ -47,7 +47,9 @@ sp_api::decl_runtime_apis! {
 	/// This api is used by the `client/authority-discovery` module to retrieve identifiers
 	/// of the current and next authority set.
 	pub trait AuthorityDiscoveryApi {
-		/// Retrieve authority identifiers of the current and next authority set.
+		/// Retrieve authority identifiers of the current authority set.
 		fn authorities() -> Vec<AuthorityId>;
+		/// Retrieve authority identifiers of the next authority set.
+		fn next_authorities() -> Vec<AuthorityId>;
```

Comment:
Please bump the version and write a migration.

Reply:
Or we keep just one method as this runtime api is authority specific and that only cares about all session keys.

No reason to have two methods.

Reply:
We only need the change on the runtime side.

Reply:
@ordian I agree with Basti - the `AuthorityDiscovery::authorities()` function can retain the same behavior of combining authorities from multiple validator sets. This is fine as long as there is a way for internal runtime code to get the actual current authorities with correct sorting.